catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/TestSolrProvider.java

```diff
@@ -498,6 +499,43 @@ public void testCreatedDates() throws Exception {
         assertEquals(MASKED_ID, createdMetacard.getSourceId());
     }
 
+    /** Tests that multivalued attributes are stored and returned
+     *
+     * @throws UnsupportedQueryException
+     * @throws IngestException
+     */
+    @Test
+    public void testCreateMultivaluedAttribute() throws UnsupportedQueryException, IngestException {
+
+        deleteAllIn(provider);
+
+        FilterFactory filterFactory = new FilterFactoryImpl();
+
+        MockMetacard metacard = new MockMetacard(Library.getFlagstaffRecord());
+        ArrayList<String> a = new ArrayList<>();
```

Comment:
Not as important for a unit test but might as well make the left hand side List instead of ArrayList for good practice and consistency.